src/OpenTelemetry.Exporter.Geneva/MsgPackExporter/MsgPackTraceExporter.cs

```diff
@@ -166,6 +166,8 @@ public ExportResult Export(in Batch<Activity> batch)
             }
         }
 
+        ExporterEventSource.Log.ExportSuccess(nameof(MsgPackTraceExporter));
```

Comment:
success depends on the `result` right?

Reply:
Since this is doing a single log, per batch - ExportCompleted is sufficient without stating success/failure.

Reply:
updated